Week2/Yash Homework/index.js

```diff
@@ -0,0 +1,99 @@
+//Strings
+
+//1
+
+const myString = "hello,this,is,a,difficult,to,read,sentence";
+
+console.log(myString);
+
+console.log(myString.length);
+
+const array = myString.split (",");
+
+console.log(array);
+
+console.log(array.join(" "));
+
+//2
+
+const str = 'dlroW olleH';
+
+console.log(str);
+
+const arr = str.split("");
+
+console.log(arr);
+
+const reverseArr = arr.reverse();
+
+console.log(reverseArr);
+
+const newStr = reverseArr.join("");
+
+console.log(newStr)
+
+
+//Arrays
+
+//1
+
+const favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+
+favoriteAnimals.push("turtle");  // adding turtle to last strings  
+
+console.log(favoriteAnimals);
+
+favoriteAnimals.splice(1,0, "eagle"); // adding eagle after first strings and no remove
+
+console.log(favoriteAnimals);
+
+favoriteAnimals.splice(3,1);
+
+console.log(favoriteAnimals);
+
+
+
+
+// Operators
```

Comment:
I think this is assignment `15`? Could you add the assingment `numbers` to before the code? otherwise it's difficult to know which code belongs to which assignment:)

Reply:
on Slack